### ISSUE TITLE
Does ROCm support Polaris 21 Family ?

### ISSUE THREAD

**Issue by HeartFire01** [xuhuisheng/rocm-gfx803#22]
Hello, I want to install ROCm on my Arch because it said capable to help model training on my machine learning program using Tensorflow, but my laptop graphics card is RX 560 (Polaris 21)...Does ROCm supported it ?..And which version are you suggest ?..Thanks

**Comment by xuhuisheng** [comment]
The RX560 is gfx803, too. So the latest version of ROCm - ROCm-5.40, can support your card RX560.
There is just a small bug in rocBLAS, you need do a small patch for rocBLAS and re-compile rocBLAS, then tensorflow won't raise NaN loss error. here is the patch : 

···
rm -rf $ROCM_GIT_DIR/rocBLAS/library/src/blas3/Tensile/Logic/asm_full/r9nano*
···

But I am afraid even we can run minst on tensorflow/pytorch, There must be some bugs for gfx803, e.g. I try to run diffustion stable on my gfx803 card, and it hanged on 100% with no reason.

**Comment by HeartFire01** [comment]
Okay i already installed ROCm 5.40 on Arch with paru, and do you know to to set PATH ROC_ENABLE_PRE_VEGA=1 ?

**Comment by xuhuisheng** [comment]
@HeartFire01 
We just need set `ROC_ENABLE_PRE_VEGA=1` as environment variable.

**Comment by HeartFire01** [comment]
> 

Sorry, i don't understand how to set ROC_ENABLE_PRE_VEGA=1 as an environment variable in Arch ?

**Comment by xuhuisheng** [comment]
I am not familiar with arch, but it is a linux operation system, so we can use `export `ROC_ENABLE_PRE_VEGA=1` to set an environment variable.
